src/main/java/pl/patrycja/ox/winnerchecker/Assessor.java

```diff
@@ -41,11 +44,16 @@ public boolean isMatchOver() {
     }
 
     @Override
-    public void newMatch(int number, Sign sign) {
+    public void newMatch(int number, Player player) {
         moves.clear();
         ui.display(board());
     }
 
+    @Override
+    public void playerHasChanged(Player player) {
+
```

Comment:
Should it be empty? Please put at least comment or TODO.
If this will stay empty then maybe you should rethink Spectator interface and usage of it.